amazon-kinesis-client/src/test/java/software/amazon/kinesis/retrieval/polling/PrefetchRecordsPublisherTest.java

```diff
@@ -377,13 +380,27 @@ record = Record.builder().data(createByteBufferWithSize(1024)).build();
     @Test(expected = IllegalStateException.class)
     public void testGetNextRecordsWithoutStarting() {
```

Comment:
oh can we please change the test name to reflect what it is doing : testSubscribeWithouStarting